samples/README.md

```diff
@@ -0,0 +1,49 @@
+### Sample programs to demonstrate SGX-LKL functionality
+
+1. Basic
+   
+   a. [Attack](basic/attack) <br>
```

Comment:
Can we really not do without HTML in this Markdown file?

Reply:
Changed the file to remove HTML